base_layer/wallet_ffi/src/lib.rs

```diff
@@ -3006,10 +3005,7 @@ pub unsafe extern "C" fn wallet_destroy(wallet: *mut TariWallet) {
     if !wallet.is_null() {
         let m = Box::from_raw(wallet);
         let l = m.shutdown();
-        match l {
-            Ok(_l) => {},
-            Err(_) => {},
-        }
+        if let Ok(_l) = l {}
```

Comment:
Both sides are..weird 🤔

Reply:
Rather use `let _ = m.shutdown();` for ln 3007 then ln 3009 to ln 3012 can be removed.

Reply:
Will give it a try